docs/integrations.md

```diff
@@ -0,0 +1,10 @@
+# Integrations
+
+- The broadway project ships with a [bundle] to use with a Symfony application.
```

Comment:
it's not shipped anymore: `There is a [bundle] available to use with your Symfony application`